TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/logic/clusters.py

```diff
@@ -1,8 +1,46 @@
-from console_link.models.cluster import Cluster
+from console_link.models.cluster import Cluster, HttpMethod
+import logging
+
+logger = logging.getLogger(__name__)
 
 
 def cat_indices(cluster: Cluster, as_json=False):
-    as_json_suffix = "?format=json" if as_json else ""
+    as_json_suffix = "?format=json" if as_json else "?v"
     cat_indices_path = f"/_cat/indices{as_json_suffix}"
     r = cluster.call_api(cat_indices_path)
     return r.json() if as_json else r.content
+
+
+def connection_check(cluster: Cluster):
+    cluster_details_path = "/"
+    caught_exception = None
+    r = None
+    try:
+        r = cluster.call_api(cluster_details_path, timeout=3)
+    except Exception as e:
+        caught_exception = e
+        logging.debug(f"Unable to access cluster: {cluster} with exception: {e}")
+    access_result = {}
+    if caught_exception is None:
+        response_json = r.json()
+        access_result['connection_message'] = "Successfully connected!"
+        access_result['cluster_version'] = response_json['version']['number']
+        access_result['connection_established'] = True
+    else:
+        access_result['connection_message'] = f"Unable to connect to cluster with error: {caught_exception}"
+        access_result['connection_established'] = False
+    return access_result
+
+
+def run_test_benchmarks(cluster: Cluster):
+    cluster.execute_benchmark_workload(workload="geonames")
+    cluster.execute_benchmark_workload(workload="http_logs")
+    cluster.execute_benchmark_workload(workload="nested")
+    cluster.execute_benchmark_workload(workload="nyc_taxis")
+
+
+# As a default we exclude system indices and searchguard indices
+def clear_indices(cluster: Cluster):
```

Comment:
Is this used anywhere? It's definitely very helpful for our testing, but I think we should put it behind a verification step where the user explicitly acknowledges that they're going to delete all the data in their cluster.

(helper for that https://click.palletsprojects.com/en/8.1.x/prompts/#confirmation-prompts)

Reply:
potentially even do something like only create a CLI command for it (or at least only enable it) if there's an env variable like `DEBUG_MODE` or `I_WANT_TO_DO_DANGEROUS_THINGS_TO_MY_CLUSTER` set

Reply:
Went ahead and added this as a CLI option, with a confirmation step added to it, thanks for the link